ui/src/clientLibraries/components/ClientJavaOverlay.tsx

```diff
@@ -17,159 +29,30 @@ const ClientJavaOverlay: FunctionComponent<{}> = () => {
-          <br />
-          <br />
-          &nbsp;&nbsp;&#125;
-          <br />
-          &#125;
-        </code>
-      </pre>
-      <h5>Using the client to execute a query</h5>
-      <pre>
-        <code>
-          String query = "from(bucket: \"my_bucket\") |> range(start: -1h)";
-          <br />
-          <br />
-          List&lt;FluxTable&gt; tables = client.getQueryApi().query(query,
-          "someorgid");
-        </code>
-      </pre>
+      <CodeSnippet copyText={initializeClientCodeSnippet} label="Java Code" />
       <h5>Writing Data</h5>
-      <p>
-        Data could be written using InfluxDB Line Protocol, Data Point or POJO
-      </p>
-      <p>
-        <b>InfluxDB Line Protocol</b>
-      </p>
-      <pre>
-        <code>
-          String data = "mem,host=host1 used_percent=23.43234543 1556896326";
-          <br />
-          <br />
-          try (WriteApi writeApi = client.getWriteApi()) &#123;
-          <br />
-          &nbsp;&nbsp;writeApi.writeRecord("bucketID", "orgID",
-          WritePrecision.NS, data);
-          <br />
-          &#125;
-        </code>
-      </pre>
-      <p>
-        <b>Data Point</b>
-      </p>
-      <pre>
-        <code>
-          Point point = Point
-          <br />
-          &nbsp;&nbsp;.measurement("mem")
-          <br />
-          &nbsp;&nbsp;.addTag("host", "host1")
-          <br />
-          &nbsp;&nbsp;.addField("used_percent", 23.43234543)
-          <br />
-          &nbsp;&nbsp;.time(1556896326L, WritePrecision.NS);
-          <br />
-          <br />
-          try (WriteApi writeApi = client.getWriteApi()) &#123;
-          <br />
-          &nbsp;&nbsp;writeApi.writePoint("bucketID", "orgID", point);
-          <br />
-          &#125;
-        </code>
-      </pre>
-      <p>
-        <b>POJO</b>
-      </p>
-      <pre>
-        <code>
-          Mem mem = new Mem();
-          <br />
-          mem.host = "host1";
-          <br />
-          mem.used_percent = 23.43234543;
-          <br />
-          mem.time = Instant.now();
-          <br />
-          <br />
-          try (WriteApi writeApi = client.getWriteApi()) &#123;
-          <br />
-          &nbsp;&nbsp;writeApi.writeMeasurement("bucketID", "orgID",
-          WritePrecision.NS, mem);
-          <br />
-          &#125;
-        </code>
-      </pre>
-      <pre>
-        <code>
-          @Measurement(name = "mem")
-          <br />
-          public class Mem &#123;
-          <br />
-          &nbsp;&nbsp;@Column(tag = true)
-          <br />
-          &nbsp;&nbsp;String host;
-          <br />
-          <br />
-          &nbsp;&nbsp;@Column
-          <br />
-          &nbsp;&nbsp;Double used_percent;
-          <br />
-          <br />
-          &nbsp;&nbsp;@Column(timestamp = true)
-          <br />
-          &nbsp;&nbsp;Instant time;
-          <br />
-          &#125;
-        </code>
-      </pre>
+      <p>Option 1: Example for writing using InfluxDB Line Protocol</p>
```

Comment:
~Option 1: Example for writing using InfluxDB Line Protocol~ -> Option 1: Use InfluxDB Line Protocol to write data